contracts/governance/extensions/GovernorVotesERC721.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts v4.3.2 (governance/extensions/ERC721GovernorVotesERC721.sol)
+
+pragma solidity ^0.8.0;
+
+import "../Governor.sol";
+import "../../token/ERC721/extensions/draft-ERC721Votes.sol";
+import "../../utils/math/Math.sol";
```

Comment:
It looks like Math.sol is not needed here (same comment apply to `governance/extensions/GovernorVotes.sol`)